### ISSUE TITLE
Incorrect content-length in callback.js when body contains non-ascii characters

### ISSUE THREAD

**Issue by NickMitin** [yjs/y-websocket#52]
Incorrect content-length in callback.js when body contains non-ascii characters

callback.js, line 46 is:
```js
     'Content-Length': data.length
```

callback.js, line 46 should be:
```js
    'Content-Length': Buffer.byteLength(data)
```

**Comment by dmonad** [comment]
You are right, the `data` object should be converted to a buffer first to determine the length of the content. 

The RFC specifies that the `content-length` is specified as the number of octets (number of bytes) and not as the length of the string.

Would you be willing to provide a PR for this? I currently don't have a lot of time to spare..

**Comment by NickMitin** [comment]
Sure, will do it in few days!

**Comment by xiongjiaojiao** [comment]
hi，when will this bug be fixed? @dmonad
